.github/workflows/build.yml

```
@@ -19,10 +19,10 @@ jobs:
         with:
           path: |
             **/node_modules
-          key: ${{ hashFiles('**/package-lock.json') }}
+          key: ${{ hashFiles('**/yarn.lock') }}
       - name: Install
         if: steps.cache-deps.outputs.cache-hit != 'true'
-        run: npm ci
+        run: yarn install
```

this is closer to npm ci
```suggestion
        run: yarn install --frozen-lockfile
```

---

It's true. Given that our only non-dev dependency is `@sentry/cli`, though, I wasn't sure it was worth the complication. I can add it if you think it's important.